CoreFoundation/URL.subproj/CFURLSessionInterface.h

```diff
@@ -0,0 +1,731 @@
+    CFURLSession_EasyCodeTELNET_OPTION_SYNTAX, // CURLE_TELNET_OPTION_SYNTAX
+    CFURLSession_EasyCodeOBSOLETE50, // CURLE_OBSOLETE50
+    CFURLSession_EasyCodePEER_FAILED_VERIFICATION, // CURLE_PEER_FAILED_VERIFICATION
+    CFURLSession_EasyCodeGOT_NOTHING, // CURLE_GOT_NOTHING
+    CFURLSession_EasyCodeSSL_ENGINE_NOTFOUND, // CURLE_SSL_ENGINE_NOTFOUND
+    CFURLSession_EasyCodeSSL_ENGINE_SETFAILED, // CURLE_SSL_ENGINE_SETFAILED
+    CFURLSession_EasyCodeSEND_ERROR, // CURLE_SEND_ERROR
+    CFURLSession_EasyCodeRECV_ERROR, // CURLE_RECV_ERROR
+    CFURLSession_EasyCodeOBSOLETE57, // CURLE_OBSOLETE57
+    CFURLSession_EasyCodeSSL_CERTPROBLEM, // CURLE_SSL_CERTPROBLEM
+    CFURLSession_EasyCodeSSL_CIPHER, // CURLE_SSL_CIPHER
+    CFURLSession_EasyCodeSSL_CACERT, // CURLE_SSL_CACERT
+    CFURLSession_EasyCodeBAD_CONTENT_ENCODING, // CURLE_BAD_CONTENT_ENCODING
+    CFURLSession_EasyCodeLDAP_INVALID_URL, // CURLE_LDAP_INVALID_URL
+    CFURLSession_EasyCodeFILESIZE_EXCEEDED, // CURLE_FILESIZE_EXCEEDED
+    CFURLSession_EasyCodeUSE_SSL_FAILED, // CURLE_USE_SSL_FAILED
+    CFURLSession_EasyCodeSEND_FAIL_REWIND, // CURLE_SEND_FAIL_REWIND
+    CFURLSession_EasyCodeSSL_ENGINE_INITFAILED, // CURLE_SSL_ENGINE_INITFAILED
+    CFURLSession_EasyCodeLOGIN_DENIED, // CURLE_LOGIN_DENIED
+    CFURLSession_EasyCodeTFTP_NOTFOUND, // CURLE_TFTP_NOTFOUND
+    CFURLSession_EasyCodeTFTP_PERM, // CURLE_TFTP_PERM
+    CFURLSession_EasyCodeREMOTE_DISK_FULL, // CURLE_REMOTE_DISK_FULL
+    CFURLSession_EasyCodeTFTP_ILLEGAL, // CURLE_TFTP_ILLEGAL
+    CFURLSession_EasyCodeTFTP_UNKNOWNID, // CURLE_TFTP_UNKNOWNID
+    CFURLSession_EasyCodeREMOTE_FILE_EXISTS, // CURLE_REMOTE_FILE_EXISTS
+    CFURLSession_EasyCodeTFTP_NOSUCHUSER, // CURLE_TFTP_NOSUCHUSER
+    CFURLSession_EasyCodeCONV_FAILED, // CURLE_CONV_FAILED
+    CFURLSession_EasyCodeCONV_REQD, // CURLE_CONV_REQD
+    CFURLSession_EasyCodeSSL_CACERT_BADFILE, // CURLE_SSL_CACERT_BADFILE
+    CFURLSession_EasyCodeREMOTE_FILE_NOT_FOUND, // CURLE_REMOTE_FILE_NOT_FOUND
+    CFURLSession_EasyCodeSSH, // CURLE_SSH
+    CFURLSession_EasyCodeSSL_SHUTDOWN_FAILED, // CURLE_SSL_SHUTDOWN_FAILED
+    CFURLSession_EasyCodeAGAIN, // CURLE_AGAIN
+    CFURLSession_EasyCodeSSL_CRL_BADFILE, // CURLE_SSL_CRL_BADFILE
+    CFURLSession_EasyCodeSSL_ISSUER_ERROR, // CURLE_SSL_ISSUER_ERROR
+    CFURLSession_EasyCodeFTP_PRET_FAILED, // CURLE_FTP_PRET_FAILED
+    CFURLSession_EasyCodeRTSP_CSEQ_ERROR, // CURLE_RTSP_CSEQ_ERROR
+    CFURLSession_EasyCodeRTSP_SESSION_ERROR, // CURLE_RTSP_SESSION_ERROR
+    CFURLSession_EasyCodeFTP_BAD_FILE_LIST, // CURLE_FTP_BAD_FILE_LIST
+    CFURLSession_EasyCodeCHUNK_FAILED, // CURLE_CHUNK_FAILED
+    CFURLSession_EasyCodeNO_CONNECTION_AVAILABLE, // CURLE_NO_CONNECTION_AVAILABLE
+    CFURLSession_EasyCodeSSL_PINNEDPUBKEYNOTMATCH, // CURLE_SSL_PINNEDPUBKEYNOTMATCH
+    CFURLSession_EasyCodeSSL_INVALIDCERTSTATUS, // CURLE_SSL_INVALIDCERTSTATUS
+} CFURLSession_EasyCode;
```

Comment:
it is worth noting that when porting to Linux these will not be exported in the same manner as enums on Darwin. We have numerous other examples that show the `#if os(OSX) || os(iOS)` re-export as constant pattern that must be done.  it might be worthwhile to modify that script that generates these to generate those as well.